README.md

```diff
@@ -1,26 +1,21 @@
-# Laravel Nova Tab [![Total Downloads](https://poser.pugx.org/arsenaltech/nova-tab/downloads)](https://packagist.org/packages/arsenaltech/nova-tab)
-
+# Laravel Nova Tab
 
 Custom Nova field to render tabs
 
-![Edit](http://take.ms/MFDjX)
-![Detail](http://take.ms/44q7f)
-![Validation](http://take.ms/S6hZI)
-
 ## Installation
 
 Install the package into a Laravel app that uses [Nova](https://nova.laravel.com) with Composer:
 
 ```bash
-composer require arsenaltech/nova-tab
+composer require 64robots/nova-tab
```

Comment:
please put this back the way it was since it doesn't comply with the changes you told you where making.
> This PR disables url fragment in order to be compatible with Nova and its router.